tree/ntuple/v7/src/RFieldBase.cxx

```diff
@@ -388,7 +388,7 @@ ROOT::Experimental::RFieldBase::Create(const std::string &fieldName, const std::
          if (arrayDef.size() != 2) {
             return R__FORWARD_RESULT(fnFail("the template list for std::array must have exactly two elements"));
          }
-         auto arrayLength = std::stoi(arrayDef[1]);
+         auto arrayLength = std::stoull(arrayDef[1]);
```

Comment:
We probably want more error checking here, for example `std::stoull("123junk") = 123` without error reporting. `std::stoull` takes a second `pos` argument that stores the number of processed characters, or we can use the underlying `std::strtoull` where `str_end` should point to `NULL` after successful conversion.

Reply:
While looking again into the integer parsing, I realized that as of C++11, unsuffixed integers are `long long`, and the suffix `u` supports ints up to `unsigned long long`. So I think we should simplify the spec and the implementation and only add `u` if the literal is larger than 2^63.